### DISCUSSION TITLE
Cross-referencing list items

### DISCUSSION THREAD

**Discussion by octogenary** [quarto-dev/quarto-cli#4045] [Feature Requests]
As documented, it is possible to cross-reference e.g. theorems, figures, &c.

In LaTeX, it is possible to `\label` list items in `enumerate`. I wonder whether analogous functionality could be developed in Quarto.

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/4944
